### ISSUE TITLE
Tutorial II issue - Using your own repo - issues bringing your part 1 repo to part 2

### ISSUE THREAD

**Issue by jacebenson** [redwoodjs/redwood#4832]
I was going through the rest of the tutorials on my windows 11 computer with the previously done part 1 of the tutorial.

Here its suggested to use the example repo as its has more even thought above it suggests we should be at the same place. 
Lets be consistent.  Either lets bring the tutorial part 1 to include thsi test suite or lets add it as part of the the tutorial part two.

There's loads of differences (so many) between the example repo and the part 1 of the tutorial.

I would love to make this simplier.  I spent 35 minutes trying to use my "part 1" repo and eventually gave up.
Can we do one of these to simplify this?

1. Updating Part 2 of the tutorial to include styling the the stuff from part 1
2. Abandoning the work done in part 1 and only refering to the example repo
3. Using some cli command to "update the repo" to match the example repo?

The problems with using your own repo is... all the file changes (I downloaded the example repo and copied ./web over from it to mine.   Here's the differences.
https://github.com/jacebenson/redwood-rc1-tutorial/commit/25c3779ff2839934bba2402d9aed14e9178610ba

**Comment by thedavidprice** [comment]
Looping in @cannikin 

Some great feedback here from @jacebenson

**Comment by cannikin** [comment]
Yeah there are several warnings in there that continuing with your own repo is going to be tough. I recently updated the text to (hopefully) make it even more clear: https://redwoodjs.com/docs/tutorial/intermission I have a later note during the first test suite run that you'll receive several errors if you stuck with your own codebase.

What could we do to let you continue to use your codebase from part 1? 

* Write test/stories from the beginning—I think this would be overwhelming for folks who are new to all of this and provides a huge hurdle to showing the fun stuff like cells and generators if you have to stop every time to write stories/tests.
* Provide all the test files that you can copy/paste over top of all of your files. Two problems with this: 1) I try really hard to never do a "just copy and paste this, trust me" thing as I think it takes you out of the experience of learning in small, incremental steps, and 2) if you made any minor "embellishments" do your code as you went through, the test suite may not match up and you'll get errors anyway
* Delete all the existing test/story files so they don't error out. This seems really dangerous, and I believe at some points we got into those existing test files and add additional tests, which would be really confusing
* Don't even provide a chance to continue with your own repo and just force a checkout of the part1 repo. This is probably the most viable option, and is strongly recommended throughout that new Intermission page. I can't *make* you delete you your current codebase, though.

I rewrote the section about the site having a great personality, it was kind of a lame joke anyway. We're also not splitting up the tutorial into parts 1/2 so that should help disambiguate the capital I for 1 and the actually letter "I".
